tests/unit/wc-payment-api/test-class-wc-payments-api-client.php

```diff
@@ -191,7 +192,16 @@ public function test_create_setup_intent() {
 			->expects( $this->once() )
 			->method( 'remote_request' )
 			->with(
-				$this->contains( 'https://public-api.wordpress.com/wpcom/v2/sites/%s/wcpay/setup_intents' ),
+				[
```

Comment:
This might be bulky to do for any new tests we add, any way to have the same URL checking functionality only like the original test?

Reply:
That `contains()` was added [here](https://github.com/Automattic/woocommerce-payments/pull/3303/files ) because of idempotency key. The creation of key was part of the mocked class, and it was impossible to recreate mocking with a value that is always random.

If you open the mentioned commit and compare it with changes from this one, you will see that they are the same. Since includes() function is no longer needed, the test worked before, should work now too.